cfpb/consumerfinance.gov
# AJAX-ify TCCP card filtering page via htmx

Auto-update the TCCP results whenever a filterable list control is changed using htmx's [boost attribute](https://htmx.org/attributes/hx-boost/).

## How to test this PR

1. `./frontend.sh`
2. Visit http://localhost:8000/consumer-tools/credit-cards/explore-cards/cards/ and change a filter.
3. The page results page should auto-reload.

## Notes

By default, htmx swaps out the page's `body` with the HTML returned from the server. For better or worse, this means the UX is identical to a full page reload caused by clicking the submit button. If we decide we don't like this UX, we can create a new django view for just the card results table and tell htmx to swap out the contents of just that container (we'd also need a loading state, visual spinner, etc.).

## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
